src/Lorentz/Contracts/BaseDAO/TZIP16Metadata.hs

```diff
@@ -0,0 +1,46 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+-- | TZIP-16 metadata.
+module Lorentz.Contracts.BaseDAO.TZIP16Metadata
+  ( knownBaseDAOMetadata
+  ) where
+
+import Lorentz hiding (View)
+import Universum (one)
+
+import Lorentz.Contracts.Spec.TZIP16Interface
+
+import Lorentz.Contracts.BaseDAO.Types
+
+-- For metadata storage does not matter
+type MetadataStorage = Storage () ()
+
+-- | Parts of metadata that can be filled just knowing the contract,
+-- without user's input.
+knownBaseDAOMetadata :: Metadata (ToT MetadataStorage)
+knownBaseDAOMetadata = mconcat
+  [ version "0.0.1"
+  , license $ License "LicenseRef-MIT-TQ" Nothing
```

Comment:
I _think_ it should be simply `MIT`. Like here: https://spdx.org/licenses/MIT.html